config/theme.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | JavaScript
+    |--------------------------------------------------------------------------
+    |
+    | Links a script file to the generated page at the right time according to
+    | the script dependencies, if the script has not been already included and
+    | if all the dependencies have been registered. You could either link a
+    | script with a handle previously registered using the wp_register_script()
+    | function, or provide this function with all the parameters necessary to
+    | link a script.
+    |
+    | @link https://developer.wordpress.org/reference/functions/wp_enqueue_script/#more-information
+    |
+    */
+
+    'scripts' => [
+        [
+            'handle'       => 'sage/vendor',
+            'src'          => 'scripts/vendor.js',
+            'dependencies' => ['jquery'],
+            'version'      => null,
+            'in_footer'    => true,
+        ],
+        [
+            'handle'       => 'sage/app',
+            'src'          => 'scripts/app.js',
+            'dependencies' => ['sage/vendor', 'jquery'],
+            'version'      => null,
+            'in_footer'    => true,
+        ]
+    ],
+
+
+    /*
+    |--------------------------------------------------------------------------
+    | Inline JavaScript
+    |--------------------------------------------------------------------------
+    |
+    | Code will only be added if the script is already in the queue.
+    | Accepts a string $data containing the Code. If two or more code blocks
+    | are added to the same script $handle, they will be printed in the order
+    | they were added, i.e. the latter added code can redeclare the previous.
+    |
+    | @link https://developer.wordpress.org/reference/functions/wp_add_inline_script/#description
+    |
+    */
+
+    'inline_scripts' => [
+        [
+            'handle'   => 'scripts/manifest',
```

Comment:
handle should be sage/vendor

Reply:
Corrected in (Commit 
8ffa414)[https://github.com/roots/sage/pull/2189/commits/8ffa414cbf1e51500545b42f3298c90991477e75]